samples/helloc++/README.md

```diff
@@ -1,5 +1,5 @@
-# C++ sample
-This sample shows how to build a confidential C++ application and run it in Marblerun. This should serve as a blueprint for making existing applications Marblerun-ready or creating new ones.
+# How to create a C++ Marble
+This sample shows how to build a confidential C++ application and run it in Marblerun. This can serve you as a blueprint for making existing applications Marblerun-ready or creating new creating new [Marbles](https://www.marblerun.sh/docs/getting-started/marbles/). If you haven't already, [setup Marblerun](../../BUILD.md#build) to get ready.
```

Comment:
```suggestion
This sample shows how to build a confidential C++ application and run it in Marblerun. This can serve you as a blueprint for making existing applications Marblerun-ready or creating new [Marbles](https://www.marblerun.sh/docs/getting-started/marbles/). If you haven't already, [setup Marblerun](../../BUILD.md#build) to get ready.
```